addons/website_event/views/event_templates_page.xml

```diff
@@ -17,9 +17,9 @@
 
 <template id="navbar" name="Event Navbar">
     <div class="container d-flex align-items-center mt-3 mb-3 mb-lg-2">
-        <a href="/event">
+        <a href="/event" title="All Events">
```

Comment:
:thinking: Did you export the .pot automatically actually? Do we not add `title` attributes to .pot for some reason? Seems fishy if so